src/interface/api-outbound.yaml

```diff
@@ -478,6 +520,17 @@ components:
             - accountNickname: dfspa.user.nickname2
               id: dfspa.username.5678
               currency: USD
+    ConsentRequestsValidateResponse:
+      description: Consent requests validate response
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              consent:
+                $ref: '#/components/schemas/ConsentsPostRequest'
```

Comment:
a Request as a Response part - it doesn't fit to me ;-)

Reply:
yea...we'll revisit this...